instrumentation/src/main/java/io/opentelemetry/android/export/SpanDataModifier.java

```diff
@@ -127,7 +118,7 @@ public <T> SpanDataModifier replaceSpanAttribute(
         return this;
     }
 
-    public SpanExporter build() {
+    public SpanExporter wrap(SpanExporter delegate) {
```

Comment:
Sorry that this is kinda unrelated to the main change, but I noticed this and decided that `wrap()` was a more fitting semantic than a build method not on a builder that returns an unrelated type.